core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -1104,12 +1100,11 @@ private void endRequest0(@Nullable Throwable requestCause, long requestEndTimeNa
             }
         }
 
-        // Set names if request content is not deferred or it was deferred but has been set
-        // before the request completion.
-        if (!hasInterestedFlags(deferredFlags, RequestLogProperty.REQUEST_CONTENT) ||
-            isAvailable(RequestLogProperty.REQUEST_CONTENT)) {
+        // Set names if request content is not deferred
+        if (!hasInterestedFlags(deferredFlags, RequestLogProperty.REQUEST_CONTENT)) {
```

Comment:
Previously, the default name was filled in if there was no custom value before a request ended. If a deferred request content is called before `endRequest()`, `RequestLogBuilder.name()` is ignored, which could be a breaking change. Would it be worth specifying this change in Javadoc?

Reply:
Sure, done